develop_server.sh

```diff
@@ -1,103 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
we don't use it... the application is run through `gulp-connect` This is just an erroneous file.